### PR TITLE
Fix score and progress reports not loading

### PR DESCRIPTION
**Pull request by Emily-ejag**
## Proposed changes

Please log in as admin --- check out the score/progress reports for recent created admins or admins where kids do not have name.

## Types of changes

What types of changes does this pull request introduce?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (non-breaking change that does not add functionality but makes code cleaner or more efficient)
- [ ] Documentation Update
- [ ] Tests (new or updated tests)
- [ ] Style (changes to code styling)
- [ ] CI (continuous integration changes)
- [ ] Repository Maintenance
- [ ] Other (please describe below)

## Checklist

- [x] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
- [x] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
- [x] Linting checks pass with my changes.
- [x] Any existing unit tests pass with my changes.
- [x] Any existing end-to-end tests pass with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
- [ ] I have added JSDoc comments as appropriate.
- [ ] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).
- [ ] I have shared this PR on the roar-pr-reviews channel (if I have access)
- [ ] I have linked relevant issues (if any)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "vue-recaptcha": "^3.0.0-alpha.6",
     "vue-router": "^4.1.6",
     "vue3-text-clamp": "^0.1.2",
+    "workbox-precaching": "^7.1.0",
```
Comment (kellyel) [inline package.json:85]: Is this related to the `workbox-precaching` error I have been seeing in local recently?

Comment (Emily-ejag) [reply package.json:85]: I think so, I had to install it. It worked for me! I hope it helps, otherwise I can take it out :)

**Review by kellyel — APPROVED** [review]

File: src/pages/ScoreReport.vue [hunk 2]
```diff
@@ -625,15 +625,15 @@ const computeAssignmentAndRunData = computed(() => {
         schoolName = schoolNameDictionary.value[schoolId];
       }
 
-      const firstNameOrUsername = user.name.first ?? user.username;
+      const firstNameOrUsername = user.name?.first ?? user.username;
```
Comment (ksmontville) [inline src/pages/ScoreReport.vue:628]:
```suggestion
      const firstName = user.name?.first;
      const lastName = user.name?.last;
      const username = user.username;
      
      const firstNameOrUsername = firstName ?? username;
```

File: src/pages/ScoreReport.vue [hunk 3]
```diff
@@ -625,15 +625,15 @@ const computeAssignmentAndRunData = computed(() => {
...
 
       const currRow = {
         user: {
           username: user.username,
           email: user.email,
           userId: user.userId,
-          firstName: user.name.first,
-          lastName: user.name.last,
+          firstName: user.name?.first,
+          lastName: user.name?.last,
```
Comment (ksmontville) [inline src/pages/ScoreReport.vue:636]:
```suggestion
          firstName: firstName;
          lastName: lastName;
```

**Review by ksmontville — APPROVED** [review]
Just some ideas, but I approve otherwise.

File: src/pages/ScoreReport.vue (hunk 2 above)
Comment (Emily-ejag) [reply src/pages/ScoreReport.vue:628]: let me try

File: src/pages/ScoreReport.vue (hunk 3 above)
Comment (Emily-ejag) [reply src/pages/ScoreReport.vue:636]: let me try x2

**Pull request merged**